src/dispatch/incident/models.py

```diff
@@ -273,10 +273,33 @@ class IncidentCreate(IncidentBase):
     tags: Optional[List[TagRead]] = []
 
 
+class IncidentReadMinimal(IncidentBase):
```

Comment:
Looks like `cases` was dropped in this version? I think that could impact our ability to display related cases in the details tab in the UI and potentially in other places.

Reply:
It shouldn't; this should only be used for bulk listing incidents. afik we only display case data when we load incidents individually. If there is a place where we display case data alongside bulk incident information, we should be wary of if we need to present it as the incident <-> case SQL join is quite expensive.

Reply:
I've tested the changes and they're causing related cases and other data needed in the different tabs in the details view to _not_ load, because we don't use `IncidentRead` when we view incident details. We use `IncidentReadMinimal`. Same is happening with cases.

<img width="851" alt="Screenshot 2023-01-09 at 10 11 47 AM" src="https://user-images.githubusercontent.com/39573146/211378043-114db972-7211-4d7a-8eb7-91b384d3ed9b.png">
<img width="600" alt="Screenshot 2023-01-09 at 10 12 22 AM" src="https://user-images.githubusercontent.com/39573146/211378246-51b05087-0fe4-4631-95eb-816c4376ddf1.png">
<img width="600" alt="Screenshot 2023-01-09 at 10 12 29 AM" src="https://user-images.githubusercontent.com/39573146/211378248-8482858d-6389-4835-a0e6-b19b6717467b.png">

Reply:
Looks like we already tried this in https://github.com/Netflix/dispatch/commit/0dacd7dec2c0bc4c33f55495cbf47086d62b0528 about a year ago and I had to swap `IncidentReadMinimal` with `IncidentRead` in https://github.com/Netflix/dispatch/commit/20b41b4cd7eb0e9afceb51109304a5d529a74f78 because we ran into the same issue.

Reply:
I think one way to fix this would be to issue an API query to get the single incident or case when we click view/ edit in the UI, which would then use the `IncidentRead` Pydantic model (https://github.com/Netflix/dispatch/blob/master/src/dispatch/incident/views.py#L86). We would still need to add `cases` to the `IncidentRead` model, and `incidents` in the `CaseRead` (missing) model?

Reply:
Okay, I'm not sure why we are not fetching an individual item on edit. Let me see if I can make that change.

Reply:
I have updated the PR to ensure that we do a complete fetch instead of relying on data from the bulk endpoint.

Reply:
Did the same for cases and changed the typing and added missing fields in the case pydantic models.